src/components/elements/buttons/index.js

```diff
@@ -0,0 +1,75 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+import Arrow from '../../../../static/svg/arrow'
+import { font, size, letterSpacing, colors } from '../../../utils'
+
+const StyledButton = styled.button`
+  height: 5rem;
+  ${font};
+  font-size: ${size.xxs};
+  font-weight: 500;
+  letter-spacing: ${letterSpacing.medium};
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 1rem 2rem;
+  border-radius: 5px;
+  border: none;
```

Comment:
I think you need to remove the outline and the cursor should be set to pointer here
```css
{
  outline: none;
  cursor: pointer;
}
```

Reply:
done